README.md

```diff
@@ -1,143 +1,415 @@
+  - [Choosing levels](#choosing-levels)
+  - [Level distribution](#level-distribution)
+  - [Activities by level](#activities-by-level)
+  - [Promotions](#promotions)
+- [Orbit KPIs](#orbit-kpis)
+- [Full Example](#full-example)
+- [Additional topics](#additional-topics)
+  - [Groups](#groups)
+  - [Orbit vs. funnel](#orbit-vs-funnel)
+- [Resources](#resources)
+- [Early adopters](#early-adopters)
+- [About the Orbit Model](#about-the-orbit-model)
+- [Contributing](#contributing)
+- [Changelog](#changelog)
+
+# Introduction
+
+The Orbit Model is a vocabulary and visual canvas that helps organizations understand, manage, and grow their community. Questions like:
+
+- How do I measure my community's engagement and growth?
+- How do I attract new people to my community?
+- Which members should I prioritize spending time with?
+- What contribution should I ask each community member to make?
+
+Who can use the Orbit Model?
+
+- Developer communities
+- Open source projects
+- Community-driven businesses
+- and more...
+
+### The solar system
+
+A solar system, with all of its celestial bodies and concentric orbits, lends itself well to visualizing a community. Some members are close to the heart of things, others far away. Some members are as large and visible as Jupiter, others barely the size of an asteroid. And of course, everything is constantly in motion.
+
+To make it easy to discuss the differences between members, so we can give them the appropriate experience, the Orbit Model assigns each person 2 core attributes: _love_ and _reach_.
+
+**The goal is to increase Love and Reach for each of your community members.**
+
+## Love
+
+Love is a member's level of engagement and investment in the community. Someone with high love is highly active, capable, and experienced. They're willing and able to play key roles in the community like contributing, moderating, and organizing.
+
+> "Jennifer has come to the last 5 meetups and next month she's hosting her own. In addition, she just published a blog post about how to install our package on Linux. Her love is increasing fast this year!"
+
+Members have different degrees of love for each of the communities that they participate in, so love is measured on a community-by-community basis. I might have high love for the running club I help organize but relatively low love for my neighborhood association where I usually skip the meetings.
+
+**Knowing the love of your community members helps you decide what opportunities you should be offering them,** ranging from those that require a large amount of time and responsibility down to simpler contributions that they can make in a few minutes.
+
+**The more inline your offer is with the community member's current engagement level, the more likely they are to take you up on it.**
+
+> On the visual canvas of the Orbit Model, love is represented by distance from the center of the solar system. The more love, the closer to the center.
```

Comment:
I was expecting another example here, since pull quotes seem to be used as illustrations/examples. Consider using an icon for these little tips, like 💡 or  🔭

Reply:
Good idea!